mne/decoding/transformer.py

```diff
@@ -715,3 +719,157 @@ def transform(self, X):
         X = self.estimator.transform(X)
         X = np.reshape(X.T, [-1, n_epochs, n_times]).transpose([1, 0, 2])
         return X
+
+
+class TemporalFilter(TransformerMixin):
+    """Estimator to filter data array along the last dimension, having at
+    least 2 dimensions. Can be applied on epochs, evoked data.
+    This is compatible with scikit-learn pipeline and therefore useful for
+    chaining MNE processing steps.
+
+    Applies a zero-phase low-pass, high-pass, band-pass, or band-stop
+    filter to the channels.
+
+    l_freq and h_freq are the frequencies below which and above which,
+    respectively, to filter out of the data. Thus the uses are:
+
+        - l_freq < h_freq: band-pass filter
+        - l_freq > h_freq: band-stop filter
+        - l_freq is not None, h_freq is None: low-pass filter
+        - l_freq is None, h_freq is not None: high-pass filter
+
+    If n_jobs > 1, more memory is required as "len(picks) * n_times"
+    additional time points need to be temporarily stored in memory.
```

Comment:
I would remove these two lines and point the user to the see the filter function.